pages/1.13/api/oss-iam.yaml

```diff
@@ -0,0 +1,316 @@
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: 'Identity and Access Management API'
+schemes:
+  - http
+  - https
+basePath: '/acs/api/v1'
+tags:
+  - name: login
+    description: Login- and authentication-related endpoints
+  - name: oidc
+    description: OpenID Connect endpoints, including provider management
+  - name: users
+    description: User management endpoints
+
+paths:
+  /auth/login:
+    post:
+      summary: Log in (obtain a DC/OS authentication token).
+      description: >
+        Exchange user credentials (regular user account: uid and password;
+        service user account: uid and service login token) for a DC/OS
+        authentication token. The resulting DC/OS authentication token is an RFC
+        7519 JSON Web Token (JWT) of type RS256. It has a limited lifetime which
+        depends on the IAM configuration (only, i.e. the lifetime cannot be
+        chosen as part of the login HTTP request). The DC/OS authentication
+        token can be verified out-of-band using a standards-compliant RS256 JWT
+        verification procedure based on the long-lived public key material
+        presented by the IAM's /auth/jwks endpoint, and by requiring the two
+        claims `exp` and `uid` to be present.
+      tags:
+        - login
+      consumes:
+        - application/json
+      produces:
+        - application/json
+      parameters:
+        - name: LoginObject
+          description: uid & password or uid & service login token.
+          in: body
+          required: true
+          schema:
+            $ref: "#/definitions/LoginObject"
+      responses:
+        200:
+          description: >
+            Login successful. The response body contains a JSON
+            object providing the authentication token.
```

Comment:
This file is a Swagger definition that lives in the Bouncer repository. It is only copied to the docs because our Bouncer API is not yet self-documenting.

In order make changes to this file one would need to make a PR against Bouncer first.